talentmap_api/fsbid/views/panel_remarks.py

```diff
@@ -128,3 +131,72 @@ def get(self, request):
         except Exception as e:
             logger.error(f"Error getting panel remarks exception: {e}")
             return Response("Error getting panel remarks", status=status.HTTP_500_INTERNAL_SERVER_ERROR)
+
+
+class CreatePanelRemark(APIView):
+    """
+    POST endpoint that allows admin or superusers to create a new panel remark.
+
+    This endpoint reaches out to the Web Services POST endpoint to create a new panel remark.
+    The Web Service endpoint requires the following JSON format to create a new remark:
+    {
+        "rmrkrccode": "string", --category code of the remark
+        "rmrkordernum": int, --order number of the remark
+        "rmrkshortdesctext": "string",  --short description of the remark
+        "rmrkmutuallyexclusiveind": "string", --Y or N
+        "rmrktext": "string", --description of the remark
+        "rmrkactiveind": "string", --Y or N
+        "rmrkcreateid": int, --hru_id of the user creating the remark
+        "rmrkcreatedate": "string", --current date and time in ISO format
+        "rmrkupdateid": int, --hru_id of the user creating the remark
+        "rmrkupdatedate": "string" --current date and time in ISO format
+    }
+    """
+    permission_classes = [isDjangoGroupMember('superuser'), isDjangoGroupMember('admin')]
+
+    def post(self, request):
+
+        # Step 1 - Get the HRU ID of the user currenlty logged in that is creating the remark
+        jwt_token = request.META['HTTP_JWT']
+        hru_id = jwt.decode(jwt_token, verify=False).get('sub')
+        # convering to int because WS post endpoint only accepts this as an int
+        hru_id_int = int(hru_id)
```

Comment:
```suggestion
        hru_id = int(jwt.decode(jwt_token, verify=False).get('sub'))
```
could you do something link this for simplicity?